Gruntfile.js

```diff
@@ -479,7 +479,51 @@ module.exports = function(grunt) {
               // see below for options. this is optional.
             }
           }
+      },
+      // Testem
+      testem: {
+        'default': {
+          src: [
+            'bower_components/qunit/qunit/qunit.js',
+            'bower_components/jquery/dist/jquery.min.js',
+            'js/jquery.easing.1.3.js',
+            'js/animation.js',
+            'js/velocity.min.js',
+            'js/hammer.min.js',
+            'js/jquery.hammer.js',
+            'js/global.js',
+            'js/collapsible.js',
+            'js/dropdown.js',
+            'js/leanModal.js',
+            'js/materialbox.js',
+            'js/parallax.js',
+            'js/tabs.js',
+            'js/tooltip.js',
+            'js/waves.js',
+            'js/toasts.js',
+            'js/sideNav.js',
+            'js/scrollspy.js',
+            'js/forms.js',
+            'js/slider.js',
+            'js/cards.js',
+            'js/pushpin.js',
+            'js/buttons.js',
+            'js/transitions.js',
+            'js/scrollFire.js',
+            'js/date_picker/picker.js',
+            'js/date_picker/picker.date.js',
+            'js/character_counter.js',
+            'tests/setup.js',
+            'tests/helpers/**/*.js',
+            'tests/**/*-test.js'
```

Comment:
Another convention borrowed from the ember.js community - All files in the `/tests` folder that end with `-test.js` will be loaded into the test runner